src/dependency-manager/spec/utils/json-schema-annotations.ts

```diff
@@ -23,6 +23,8 @@ export const AnyOf = (...args: any): DecoratorSchema => {
       anyOf.push({
         $ref: `${REF_PREFIX}${arg.name}`,
       });
+    } else if (typeof arg === 'object' && 'type' in arg && 'pattern' in arg) {
+      anyOf.push(arg);
```

Comment:
This change was needed to specifically support this -
```ts
AnyOf({
  type: 'string',
  pattern: Slugs.ComponentTagValidator.source,
}, DependencySpec)
```

Currently you can only `AnyOf` _any_ string/object/etc, but not a string with a specific pattern, which this now fixes